[2023-12-19T19:30:14.715+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: safetyWarning_etl_storeToPostgres.create_safetyWarning_table manual__2023-12-19T19:30:02.000496+00:00 [queued]>
[2023-12-19T19:30:14.723+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: safetyWarning_etl_storeToPostgres.create_safetyWarning_table manual__2023-12-19T19:30:02.000496+00:00 [queued]>
[2023-12-19T19:30:14.752+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-19T19:30:14.763+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_safetyWarning_table> on 2023-12-19 19:30:02.000496+00:00
[2023-12-19T19:30:14.770+0000] {standard_task_runner.py:57} INFO - Started process 15521 to run task
[2023-12-19T19:30:14.773+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'safetyWarning_etl_storeToPostgres', 'create_safetyWarning_table', 'manual__2023-12-19T19:30:02.000496+00:00', '--job-id', '467', '--raw', '--subdir', 'DAGS_FOLDER/safetyWarning_etl_storeToPostgres.py', '--cfg-path', '/tmp/tmp1xi2z55p']
[2023-12-19T19:30:14.776+0000] {standard_task_runner.py:85} INFO - Job 467: Subtask create_safetyWarning_table
[2023-12-19T19:30:14.819+0000] {task_command.py:416} INFO - Running <TaskInstance: safetyWarning_etl_storeToPostgres.create_safetyWarning_table manual__2023-12-19T19:30:02.000496+00:00 [running]> on host f97daee24bd9
[2023-12-19T19:30:14.888+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='safetyWarning_etl_storeToPostgres' AIRFLOW_CTX_TASK_ID='create_safetyWarning_table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T19:30:02.000496+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-19T19:30:02.000496+00:00'
[2023-12-19T19:30:14.890+0000] {sql.py:274} INFO - Executing: 

            CREATE TABLE safety_warning (
                id SERIAL PRIMARY KEY,
                地點位置 VARCHAR(255),
                管轄警察局 VARCHAR(255),
                分局 VARCHAR(255),
                聯繫窗口 VARCHAR(255),
                聯繫電話 VARCHAR(20),
                經緯度 VARCHAR(50),
                經度 FLOAT,
                緯度 FLOAT
            );

            
[2023-12-19T19:30:14.898+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T19:30:14.906+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T19:30:14.943+0000] {sql.py:418} INFO - Running statement: 

            CREATE TABLE safety_warning (
                id SERIAL PRIMARY KEY,
                地點位置 VARCHAR(255),
                管轄警察局 VARCHAR(255),
                分局 VARCHAR(255),
                聯繫窗口 VARCHAR(255),
                聯繫電話 VARCHAR(20),
                經緯度 VARCHAR(50),
                經度 FLOAT,
                緯度 FLOAT
            );

            , parameters: None
[2023-12-19T19:30:14.963+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=safetyWarning_etl_storeToPostgres, task_id=create_safetyWarning_table, execution_date=20231219T193002, start_date=20231219T193014, end_date=20231219T193014
[2023-12-19T19:30:14.985+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-19T19:30:15.002+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
